apps/chat/src/components/Chat/TalkTo/TalkToCard.tsx

```diff
@@ -225,7 +225,7 @@ export const TalkToCard = ({
         Icon: UnpublishIcon,
         onClick: (e: React.MouseEvent) => {
           e.stopPropagation();
-          onPublish?.(entity, PublishActions.DELETE);
+          onPublish(entity);
```

Comment:
Why does unpublish menu option execute `onPublish` (ADD action as result)?

Reply:
because it's publish request anyway

Reply:
Because we agreed on hide unpublish option from talk to cards

Reply:
option removed